files/zh-cn/web/css/_colon_blank/index.html

```diff
@@ -6,7 +6,7 @@
   - CSS 选择器
   - 伪类
 translation_of: Web/CSS/:blank
-original_slug: Web/CSS/:blank空白伪类
+original_slug: Web/CSS/:blank 空白伪类
```

Comment:
this should not be modified

```suggestion
original_slug: Web/CSS/:blank空白伪类
```